Article/Article.js

```diff
@@ -26,4 +27,8 @@ class Article {
 
 */
 
-let articles;
\ No newline at end of file
+let articles = document.querySelectorAll(".article");;
+articles.forEach( article => {
```

Comment:
Good use of advanced looping function, however, I would like to see you push it a little further and make this a one-liner arrow function. :)